### ISSUE TITLE
False Crash

### ISSUE THREAD

**Issue by 2V3EvG4LMJFdRe** [iina/iina#4260]
**System and IINA version:**

- macOS 10.13.6
- IINA 1.3.1 Build 133

**Expected behavior:**

Closing IINA with command+q closes IINA

**Actual behavior:**

Closing IINA with command+q closes IINA and reports a crash with the bearing "IINA quit unexpectedly" (cut short):

```
Process:               IINA [11599]
Path:                  /Applications/IINA.app/Contents/MacOS/IINA
Identifier:            com.colliderli.iina
Version:               1.3.1 (133)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           IINA [11599]
User ID:               501

Date/Time:             2023-03-06 03:08:42.267 -0300
OS Version:            Mac OS X 10.13.6 (17G14042)
Report Version:        12
Anonymous UUID:        27D01FED-281B-D75A-D33D-303BE5AA0BCA


Time Awake Since Boot: 30000 seconds

System Integrity Protection: disabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000048
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [0]

VM Regions Near 0x48:
--> 
    __TEXT                 0000000106c9b000-0000000106f6f000 [ 2896K] r-x/r-x SM=COW  `- [/Applications/IINA.app/Contents/MacOS/IINA]

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libmpv.1.dylib                	0x000000010753ec1e mpv_get_property + 14
1   com.colliderli.iina           	0x0000000106e48418 $s4IINA10PlayerCoreC6syncUIyyAC12SyncUIOptionOF + 584
2   com.colliderli.iina           	0x0000000106e480c3 $s4IINA10PlayerCoreC10syncUITimeyyF + 51
3   com.colliderli.iina           	0x0000000106d41f51 $s4IINA20MainWindowControllerC6showUI33_656F66F44299633CE9F07D3F2FD985C2LLyyF + 385
4   com.colliderli.iina           	0x0000000106d3d3b3 $s4IINA20MainWindowControllerC23windowDidExitFullScreenyy10Foundation12NotificationVF + 595
5   com.colliderli.iina           	0x0000000106d417cd $s4IINA20MainWindowControllerC15windowWillCloseyy10Foundation12NotificationVFToTm + 109
6   com.apple.CoreFoundation      	0x00007fff5302fb5c __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
7   com.apple.CoreFoundation      	0x00007fff5302fa2a _CFXRegistrationPost + 458
8   com.apple.CoreFoundation      	0x00007fff5302f761 ___CFXNotificationPost_block_invoke + 225
9   com.apple.CoreFoundation      	0x00007fff52fed540 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1664
10  com.apple.CoreFoundation      	0x00007fff52fec677 _CFXNotificationPost + 599
11  com.apple.Foundation          	0x00007fff55100047 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
12  com.apple.AppKit              	0x00007fff5112b9de -[NSWindow(NSFullScreen) _didExitFullScreen] + 439
13  com.apple.AppKit              	0x00007fff50c34282 -[_NSWindowExitFullScreenTransitionController doAfterExitFullScreen] + 57
14  com.apple.AppKit              	0x00007fff50f1a9f6 -[_NSExitFullScreenTransitionController _doAfterExitFullScreen] + 185
15  com.apple.AppKit              	0x00007fff50f1af38 -[_NSExitFullScreenTransitionController _doSucceededToExitFullScreen] + 458
16  com.apple.AppKit              	0x00007fff50982779 __NSFullScreenDockConnectionSendExitForSpace_block_invoke + 88
17  libxpc.dylib                  	0x00007fff7b31d4f5 _xpc_connection_reply_callout + 36
18  libxpc.dylib                  	0x00007fff7b31d460 _xpc_connection_call_reply_async + 82
19  libdispatch.dylib             	0x00007fff7af9b92c _dispatch_client_callout3 + 8
20  libdispatch.dylib             	0x00007fff7afaab0a _dispatch_mach_msg_async_reply_invoke + 360
21  libdispatch.dylib             	0x00007fff7af9a221 _dispatch_main_queue_callback_4CF + 776
22  com.apple.CoreFoundation      	0x00007fff530588d9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
23  com.apple.CoreFoundation      	0x00007fff5301aa7a __CFRunLoopRun + 2586
24  com.apple.CoreFoundation      	0x00007fff53019dd3 CFRunLoopRunSpecific + 483
25  com.apple.HIToolbox           	0x00007fff52304d96 RunCurrentEventLoopInMode + 286
26  com.apple.HIToolbox           	0x00007fff52304b06 ReceiveNextEventCommon + 613
27  com.apple.HIToolbox           	0x00007fff52304884 _BlockUntilNextEventMatchingListInModeWithFilter + 64
28  com.apple.AppKit              	0x00007fff505b1a3b _DPSNextEvent + 2085
29  com.apple.AppKit              	0x00007fff50d47e34 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 3044
30  com.apple.AppKit              	0x00007fff507935dc -[NSApplication _shouldTerminate] + 1367
31  com.apple.AppKit              	0x00007fff50792ad2 -[NSApplication terminate:] + 818
32  com.apple.AppKit              	0x00007fff50974008 -[NSApplication _terminateFromSender:askIfShouldTerminate:saveWindows:] + 289
33  com.apple.AppKit              	0x00007fff5090ea8b __79-[NSApplication(NSWindowCache) _scheduleCheckForTerminateAfterLastWindowClosed]_block_invoke + 102
34  com.apple.CoreFoundation      	0x00007fff53023c94 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 20
35  com.apple.CoreFoundation      	0x00007fff53023907 __CFRunLoopDoTimer + 1095
36  com.apple.CoreFoundation      	0x00007fff5302340a __CFRunLoopDoTimers + 346
37  com.apple.CoreFoundation      	0x00007fff5301a9db __CFRunLoopRun + 2427
38  com.apple.CoreFoundation      	0x00007fff53019dd3 CFRunLoopRunSpecific + 483
39  com.apple.HIToolbox           	0x00007fff52304d96 RunCurrentEventLoopInMode + 286
40  com.apple.HIToolbox           	0x00007fff52304b06 ReceiveNextEventCommon + 613
41  com.apple.HIToolbox           	0x00007fff52304884 _BlockUntilNextEventMatchingListInModeWithFilter + 64
42  com.apple.AppKit              	0x00007fff505b1a3b _DPSNextEvent + 2085
43  com.apple.AppKit              	0x00007fff50d47e34 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 3044
44  com.apple.AppKit              	0x00007fff505a684d -[NSApplication run] + 764
45  com.apple.AppKit              	0x00007fff50575a3a NSApplicationMain + 804
46  com.colliderli.iina           	0x0000000106ca2859 main + 9
47  libdyld.dylib                 	0x00007fff7afc8015 start + 1

Thread 1:
0   libsystem_kernel.dylib        	0x00007fff7b11928a __workq_kernreturn + 10
1   libsystem_pthread.dylib       	0x00007fff7b2e020e _pthread_wqthread + 1552
2   libsystem_pthread.dylib       	0x00007fff7b2dfbe9 start_wqthread + 13

Thread 2:
0   libsystem_kernel.dylib        	0x00007fff7b11928a __workq_kernreturn + 10
1   libsystem_pthread.dylib       	0x00007fff7b2e0009 _pthread_wqthread + 1035
2   libsystem_pthread.dylib       	0x00007fff7b2dfbe9 start_wqthread + 13

Thread 3:
0   libsystem_kernel.dylib        	0x00007fff7b11928a __workq_kernreturn + 10
1   libsystem_pthread.dylib       	0x00007fff7b2e0009 _pthread_wqthread + 1035
2   libsystem_pthread.dylib       	0x00007fff7b2dfbe9 start_wqthread + 13

Thread 4:
0   libsystem_kernel.dylib        	0x00007fff7b10f24e semaphore_timedwait_trap + 10
1   libdispatch.dylib             	0x00007fff7af9e94f _dispatch_sema4_timedwait + 72
2   libdispatch.dylib             	0x00007fff7af9694e _dispatch_semaphore_wait_slow + 58
3   libdispatch.dylib             	0x00007fff7af91e53 _dispatch_worker_thread + 251
4   libsystem_pthread.dylib       	0x00007fff7b2e0661 _pthread_body + 340
5   libsystem_pthread.dylib       	0x00007fff7b2e050d _pthread_start + 377
6   libsystem_pthread.dylib       	0x00007fff7b2dfbf9 thread_start + 13

Thread 5:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	0x00007fff7b10f1fa mach_msg_trap + 10
1   libsystem_kernel.dylib        	0x00007fff7b10e714 mach_msg + 60
2   com.apple.CoreFoundation      	0x00007fff5301b405 __CFRunLoopServiceMachPort + 341
3   com.apple.CoreFoundation      	0x00007fff5301a757 __CFRunLoopRun + 1783
4   com.apple.CoreFoundation      	0x00007fff53019dd3 CFRunLoopRunSpecific + 483
5   com.apple.AppKit              	0x00007fff506eefc4 _NSEventThread + 184
6   libsystem_pthread.dylib       	0x00007fff7b2e0661 _pthread_body + 340
7   libsystem_pthread.dylib       	0x00007fff7b2e050d _pthread_start + 377
8   libsystem_pthread.dylib       	0x00007fff7b2dfbf9 thread_start + 13

Thread 6:
0   libsystem_pthread.dylib       	0x00007fff7b2dfbdc start_wqthread + 0
1   ???                           	0x000070000dfecb60 0 + 123145537112928

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x000060c00011f6e0  rbx: 0x0000000000000000  rcx: 0x00007ffee8f5fa30  rdx: 0x0000000000000005
  rdi: 0x0000000000000000  rsi: 0x0000000106f48761  rbp: 0x00007ffee8f5f990  rsp: 0x00007ffee8f5f940
   r8: 0xe700000000000000   r9: 0x0000000000000001  r10: 0x0000000000000004  r11: 0x0000000000000004
  r12: 0x000060800022c660  r13: 0x000060c0001d13a0  r14: 0x0000000000000000  r15: 0x000060c0001d13a0
  rip: 0x000000010753ec1e  rfl: 0x0000000000010202  cr2: 0x0000000000000048
  
Logical CPU:     2
Error Code:      0x00000004
Trap Number:     14


Binary Images:
       0x106c9b000 -        0x106f6eff7 +com.colliderli.iina (1.3.1 - 133) <646D0D36-CC27-3F7E-8B52-001B3772F549> /Applications/IINA.app/Contents/MacOS/IINA
       0x107431000 -        0x10747cff7 +org.sparkle-project.Sparkle (1.27.1 - 1.106) <0CBFC38A-4096-3990-B77B-E01BD3D57B2A> /Applications/IINA.app/Contents/Frameworks/Sparkle.framework/Versions/A/Sparkle
       0x1074be000 -        0x1077b9ffb +libmpv.1.dylib (0) <2D4033F1-6FAB-385F-A486-B5C1ED20280C> /Applications/IINA.app/Contents/Frameworks/libmpv.1.dylib
       0x10784e000 -        0x108255fe7 +libavcodec.58.dylib (0) <FAA5B570-1C50-3BC4-94B7-A67388E7C5E3> /Applications/IINA.app/Contents/Frameworks/libavcodec.58.dylib
       0x1088d6000 -        0x108a71fff +libavformat.58.dylib (0) <BCDDFE59-3443-3BB9-BA45-E70FDCF44C36> /Applications/IINA.app/Contents/Frameworks/libavformat.58.dylib
       0x108ac8000 -        0x108b1bfef +libavutil.56.dylib (0) <5B1E9157-5272-3640-A460-9D307E89F787> /Applications/IINA.app/Contents/Frameworks/libavutil.56.dylib
       0x108d4c000 -        0x108dcbfef +libswscale.5.dylib (0) <1B0A2F08-28A5-3F32-9CF1-A870250D2135> /Applications/IINA.app/Contents/Frameworks/libswscale.5.dylib
       0x108de4000 -        0x108df3fff +libswiftAVFoundation.dylib (5.0.1 - 1001.0.82.4) <8A6480E9-8C6C-3967-8E70-50045346944C> /Applications/IINA.app/Contents/Frameworks/libswiftAVFoundation.dylib
       0x108e06000 -        0x108e1aff7 +libswiftAppKit.dylib (5.0.1 - 1001.0.82.4) <1297109A-D760-3C74-A36C-C0273A7217EC> /Applications/IINA.app/Contents/Frameworks/libswiftAppKit.dylib
       0x108e3e000 -        0x108e43fff +libswiftCoreAudio.dylib (5.0.1 - 1001.0.82.4) <A33E78BC-49E9-358B-8253-8B04D7406BB6> /Applications/IINA.app/Contents/Frameworks/libswiftCoreAudio.dylib
       0x108e51000 -        0x108e59ff7 +libswiftCoreData.dylib (5.0.1 - 1001.0.82.4) <72A6FD8B-034C-3F50-9FF6-9D5723920201> /Applications/IINA.app/Contents/Frameworks/libswiftCoreData.dylib
       0x108e6b000 -        0x108e6efff +libswiftCoreFoundation.dylib (5.0.1 - 1001.0.82.4) <C0D551D8-F4D4-3B6F-8148-05EE1CFFC97F> /Applications/IINA.app/Contents/Frameworks/libswiftCoreFoundation.dylib
       0x108e78000 -        0x108e87fff +libswiftCoreGraphics.dylib (5.0.1 - 1001.0.82.4) <4C9FA634-B14A-343E-9F76-5AAFCA5FC077> /Applications/IINA.app/Contents/Frameworks/libswiftCoreGraphics.dylib
       0x108eab000 -        0x108eb0ffb +libswiftCoreImage.dylib (5.0.1 - 1001.0.82.4) <E2D50227-8FAE-3104-88F0-6D8E01A9AB18> /Applications/IINA.app/Contents/Frameworks/libswiftCoreImage.dylib
       0x108eba000 -        0x108ec4fff +libswiftCoreMedia.dylib (5.0.1 - 1001.0.82.4) <C6558E0E-DA51-3E76-9F00-7177819A89FB> /Applications/IINA.app/Contents/Frameworks/libswiftCoreMedia.dylib
       0x108ed2000 -        0x108ed9ff7 +libswiftDarwin.dylib (5.0.1 - 1001.0.82.4) <144DA4C8-8322-3E6E-942D-9976CAFA8BAE> /Applications/IINA.app/Contents/Frameworks/libswiftDarwin.dylib
       0x108eeb000 -        0x108f03ff7 +libswiftDispatch.dylib (5.0.1 - 1001.0.82.4) <0A4818EF-BCD8-3183-9D34-5F8D4920C05C> /Applications/IINA.app/Contents/Frameworks/libswiftDispatch.dylib
       0x108f41000 -        0x1090d9ff7 +libswiftFoundation.dylib (5.0.1 - 1001.0.82.4) <868B3228-5A2C-3556-A77A-695000EFCE27> /Applications/IINA.app/Contents/Frameworks/libswiftFoundation.dylib
       0x10925c000 -        0x10925fff3 +libswiftIOKit.dylib (5.0.1 - 1001.0.82.4) <75FE0421-8424-38AD-9372-17A730C44643> /Applications/IINA.app/Contents/Frameworks/libswiftIOKit.dylib
       0x109269000 -        0x109272fff +libswiftMetal.dylib (5.0.1 - 1001.0.82.4) <A2ACD465-EF03-3F27-B6F9-390A6D66A040> /Applications/IINA.app/Contents/Frameworks/libswiftMetal.dylib
       0x109281000 -        0x109285ff7 +libswiftObjectiveC.dylib (5.0.1 - 1001.0.82.4) <1A3E8F91-FF6E-3733-B1AD-CC4D4F3378F4> /Applications/IINA.app/Contents/Frameworks/libswiftObjectiveC.dylib
       0x109293000 -        0x109299ff7 +libswiftQuartzCore.dylib (5.0.1 - 1001.0.82.4) <FB7CCE82-C7FA-3E50-904D-2FB4006AC1DF> /Applications/IINA.app/Contents/Frameworks/libswiftQuartzCore.dylib
       0x1092a3000 -        0x1092a6fff +libswiftXPC.dylib (5.0.1 - 1001.0.82.4) <D0199D63-5E28-3E52-907E-3832FBB94FCF> /Applications/IINA.app/Contents/Frameworks/libswiftXPC.dylib
       0x1092b0000 -        0x1092d4ff7 +libswiftsimd.dylib (5.0.1 - 1001.0.82.4) <62F88382-9C77-3A6B-8F64-971EE6586554> /Applications/IINA.app/Contents/Frameworks/libswiftsimd.dylib
       0x109300000 -        0x109317fef +libswresample.3.dylib (0) <049537A6-C587-3E19-8A11-2A732869EE3F> /Applications/IINA.app/Contents/Frameworks/libswresample.3.dylib
       0x109324000 -        0x109357ff7 +libmujs.dylib (0) <BE1EC308-97F0-3D63-BF92-8CAA6DD16582> /Applications/IINA.app/Contents/Frameworks/libmujs.dylib
       0x109368000 -        0x109397ff7 +libjpeg.9.dylib (0) <312CD31D-CCCA-3C59-9A7D-CCF14CF31A97> /Applications/IINA.app/Contents/Frameworks/libjpeg.9.dylib
       0x1093a8000 -        0x1093bffff +libavdevice.58.dylib (0) <AFAF88E4-09CD-380F-AE84-FEA2D306CDA2> /Applications/IINA.app/Contents/Frameworks/libavdevice.58.dylib
       0x1093d8000 -        0x109422adf  dyld (551.5) <CB9BFB56-4511-36F1-A546-891FF770C01C> /usr/lib/dyld
       0x109476000 -        0x1097e7fff +libswiftCore.dylib (5.0.1 - 1001.0.82.4) <44D65143-94D5-306A-8D56-15359A7FF04E> /Applications/IINA.app/Contents/Frameworks/libswiftCore.dylib
       0x109ac8000 -        0x109d07fef +libavfilter.7.dylib (0) <7A5F4983-005A-3C25-BC11-1D777D46E1E5> /Applications/IINA.app/Contents/Frameworks/libavfilter.7.dylib
       0x109db4000 -        0x109df3fff +liblcms2.2.dylib (0) <14084D78-30FB-36A0-B35D-F8AAE6D18C91> /Applications/IINA.app/Contents/Frameworks/liblcms2.2.dylib
       0x109e10000 -        0x109e97ff7 +libarchive.13.dylib (0) <C6753A4A-4DF6-34AB-859C-78CB049E8A5B> /Applications/IINA.app/Contents/Frameworks/libarchive.13.dylib
       0x109ebc000 -        0x109ee7feb +libass.9.dylib (0) <B8D0B3F7-4779-374E-AA07-6DDFB572E41B> /Applications/IINA.app/Contents/Frameworks/libass.9.dylib
       0x109efc000 -        0x109f33fff +libbluray.2.dylib (0) <6CC1C682-A672-3D1B-A3C3-BE5591A3F89A> /Applications/IINA.app/Contents/Frameworks/libbluray.2.dylib
       0x109f48000 -        0x109fafff7 +libluajit-5.1.2.dylib (0) <AD5BE687-0F31-3B8D-91CF-40D0157E3BA5> /Applications/IINA.app/Contents/Frameworks/libluajit-5.1.2.dylib
       0x109fcc000 -        0x109feffff +librubberband.2.dylib (0) <43FC2BCF-199F-3CDF-BB70-58120B922356> /Applications/IINA.app/Contents/Frameworks/librubberband.2.dylib
       0x10a00c000 -        0x10a033ff3 +libuchardet.0.dylib (0) <DAF41BEA-A6CA-3DC6-8D6F-28B500640AE2> /Applications/IINA.app/Contents/Frameworks/libuchardet.0.dylib
       0x10a044000 -        0x10a0e7ff3 +libzimg.2.dylib (0) <7D6B8C28-30F2-3C77-A764-3E1D2E6779ED> /Applications/IINA.app/Contents/Frameworks/libzimg.2.dylib
       0x10a124000 -        0x10a13fff7 +liblzma.5.dylib (0) <34745817-95E0-39CD-AD3A-6CEA56006721> /Applications/IINA.app/Contents/Frameworks/liblzma.5.dylib
       0x10a150000 -        0x10a27bfef +libdav1d.5.dylib (0) <FDDF8DCC-FB09-316A-AB99-0F7A6F887849> /Applications/IINA.app/Contents/Frameworks/libdav1d.5.dylib
       0x10a34c000 -        0x10a353ffb +libsnappy.1.dylib (0) <64238755-6AD8-3398-9596-BD06DE98FB74> /Applications/IINA.app/Contents/Frameworks/libsnappy.1.dylib
       0x10a360000 -        0x10a373ffb +libspeex.1.dylib (0) <3090DB4E-D35E-398D-A3A9-C1CF375D5BE7> /Applications/IINA.app/Contents/Frameworks/libspeex.1.dylib
       0x10a380000 -        0x10a3a7fe3 +libsoxr.0.dylib (0) <86E71EC3-D7B4-3CBC-8618-B5409A49E56E> /Applications/IINA.app/Contents/Frameworks/libsoxr.0.dylib
       0x10a3ec000 -        0x10a56fffb +libgnutls.30.dylib (0) <7DE71D0E-170F-3CCE-B209-8933DFC28853> /Applications/IINA.app/Contents/Frameworks/libgnutls.30.dylib
       0x10a5c8000 -        0x10a623ff3 +libzmq.5.dylib (0) <EA51935C-9106-38CA-98A3-6588D0320FDF> /Applications/IINA.app/Contents/Frameworks/libzmq.5.dylib
       0x10a664000 -        0x10a697ff7 +libfontconfig.1.dylib (0) <DE80A7D7-DF4F-31ED-B09E-317D9C13FB63> /Applications/IINA.app/Contents/Frameworks/libfontconfig.1.dylib
       0x10a6ac000 -        0x10a72fff7 +libfreetype.6.dylib (0) <3FC45025-7707-339C-98EB-0AA7F42C560F> /Applications/IINA.app/Contents/Frameworks/libfreetype.6.dylib
       0x10a754000 -        0x10a75ffff +libintl.8.dylib (0) <BE22DF06-563B-386B-946A-D95359A9079D> /Applications/IINA.app/Contents/Frameworks/libintl.8.dylib
       0x10a76c000 -        0x10a78fffb +libpng16.16.dylib (0) <8DB5C357-2CC6-3DF6-B983-A0DAB405D35E> /Applications/IINA.app/Contents/Frameworks/libpng16.16.dylib
       0x10a7a0000 -        0x10a843ff7 +libp11-kit.0.dylib (0) <3E31D016-96A5-34A0-8E2B-6D8165396EAD> /Applications/IINA.app/Contents/Frameworks/libp11-kit.0.dylib
       0x10a898000 -        0x10a8b7ffb +libidn2.0.dylib (0) <C2DDEDCE-C0AC-3C79-99CA-7FAEF079CD54> /Applications/IINA.app/Contents/Frameworks/libidn2.0.dylib
       0x10a8c4000 -        0x10aa63ffb +libunistring.2.dylib (0) <FCF789F0-378E-32F6-A219-A8E0301184AD> /Applications/IINA.app/Contents/Frameworks/libunistring.2.dylib
       0x10aa88000 -        0x10aa97ff7 +libtasn1.6.dylib (0) <006459CE-2AF6-3BD4-9D4B-43060E96F05E> /Applications/IINA.app/Contents/Frameworks/libtasn1.6.dylib
       0x10aaa4000 -        0x10aad3ff7 +libnettle.8.dylib (0) <9F378648-17BA-379B-BA6F-3173A01E3A8A> /Applications/IINA.app/Contents/Frameworks/libnettle.8.dylib
       0x10aaec000 -        0x10ab23ff7 +libhogweed.6.dylib (0) <96800A63-AB3B-3925-B8ED-A8F6C38DE813> /Applications/IINA.app/Contents/Frameworks/libhogweed.6.dylib
       0x10ab38000 -        0x10ab97ff7 +libgmp.10.dylib (0) <B4CF0081-325E-3D7D-B3FF-272346A6A18F> /Applications/IINA.app/Contents/Frameworks/libgmp.10.dylib
       0x10abac000 -        0x10abb3f2f +libffi.8.dylib (0) <5904FE01-4E17-3EFF-A242-5CE0F8061F82> /Applications/IINA.app/Contents/Frameworks/libffi.8.dylib
       0x10abc0000 -        0x10abfffff +libsodium.23.dylib (0) <A55B902F-952E-3B46-B7C1-8CCAAC57AC17> /Applications/IINA.app/Contents/Frameworks/libsodium.23.dylib
       0x10ac18000 -        0x10ac37fff +libpostproc.55.dylib (0) <59CD3243-A25C-3EA6-9FF4-633A8B874E10> /Applications/IINA.app/Contents/Frameworks/libpostproc.55.dylib
       0x10ac44000 -        0x10ae2ffef +libtesseract.5.dylib (0) <964E1B16-42CF-38A1-BB22-D46D12C78968> /Applications/IINA.app/Contents/Frameworks/libtesseract.5.dylib
       0x10af04000 -        0x10af0fff3 +libvidstab.1.1.dylib (0) <B98B83BE-CDD5-37FD-8117-29D357B290AE> /Applications/IINA.app/Contents/Frameworks/libvidstab.1.1.dylib
       0x10af1c000 -        0x10b0efff7 +liblept.5.dylib (0) <3E5AAD14-7182-35A4-A9DE-327CF24ED635> /Applications/IINA.app/Contents/Frameworks/liblept.5.dylib
       0x10b150000 -        0x10b157fff +libgif.dylib (0) <A90EF064-EBDB-31FE-B835-080206A78E88> /Applications/IINA.app/Contents/Frameworks/libgif.dylib
       0x10b164000 -        0x10b1c3ff7 +libtiff.5.dylib (0) <624E90FA-D08E-3F17-BEB7-A6A25190610D> /Applications/IINA.app/Contents/Frameworks/libtiff.5.dylib
       0x10b1dc000 -        0x10b1e3ff7 +libwebpmux.3.dylib (0) <874A004E-F2EC-3A7D-B906-2E21D751560F> /Applications/IINA.app/Contents/Frameworks/libwebpmux.3.dylib
       0x10b1f0000 -        0x10b243ff3 +libwebp.7.dylib (0) <3B593048-D6A1-35D3-9B90-DD363D5B870F> /Applications/IINA.app/Contents/Frameworks/libwebp.7.dylib
       0x10b25c000 -        0x10b2a3ffb +libopenjp2.7.dylib (0) <91E0773C-291B-3E2B-B1F8-5B7F60783C43> /Applications/IINA.app/Contents/Frameworks/libopenjp2.7.dylib
       0x10b2b8000 -        0x10b36ffdb +libzstd.1.dylib (0) <79DAC82D-7CE1-343D-A86D-0D0EC93A6EC7> /Applications/IINA.app/Contents/Frameworks/libzstd.1.dylib
       0x10b38c000 -        0x10b3abff7 +liblz4.1.dylib (0) <A0B63EAB-C4F6-3044-BB49-F884DC2AF5E9> /Applications/IINA.app/Contents/Frameworks/liblz4.1.dylib
       0x10b3b8000 -        0x10b3cffff +libb2.1.dylib (0) <DEDF02AC-51C2-37E6-B888-96152E6F9CD6> /Applications/IINA.app/Contents/Frameworks/libb2.1.dylib
       0x10b3dc000 -        0x10b3f7ff7 +libfribidi.0.dylib (0) <75A1B0B1-F8A7-3585-8CF6-2D1B698659F2> /Applications/IINA.app/Contents/Frameworks/libfribidi.0.dylib
       0x10b404000 -        0x10b4b3ff7 +libharfbuzz.0.dylib (0) <4D24F082-EBFD-393F-A338-157CDE816815> /Applications/IINA.app/Contents/Frameworks/libharfbuzz.0.dylib
       0x10b50c000 -        0x10b523fff +libgraphite2.3.dylib (0) <0C8D6EC5-6F96-32D9-B651-CAA9112B2550> /Applications/IINA.app/Contents/Frameworks/libgraphite2.3.dylib
       0x10b534000 -        0x10b613ff7 +libglib-2.0.0.dylib (0) <8BD57F9E-C18F-38DC-806B-8ACCBDECFF34> /Applications/IINA.app/Contents/Frameworks/libglib-2.0.0.dylib
       0x10b648000 -        0x10b6b3ff3 +libpcre.1.dylib (0) <B595ED0D-2872-33D0-B056-98769A4DC907> /Applications/IINA.app/Contents/Frameworks/libpcre.1.dylib
       0x10b6c4000 -        0x10b6d7fff +libxcb.1.dylib (0) <CA4A6623-F595-3864-A8C5-2A151EFAA054> /Applications/IINA.app/Contents/Frameworks/libxcb.1.dylib
       0x10b6f0000 -        0x10b6f3fff +libxcb-shm.0.dylib (0) <C4C4CBD9-D28C-331C-9C83-259EC06CDF9F> /Applications/IINA.app/Contents/Frameworks/libxcb-shm.0.dylib
       0x10b700000 -        0x10b703fff +libxcb-shape.0.dylib (0) <4E7B71A1-7719-34A8-B96A-6613A2D80F7C> /Applications/IINA.app/Contents/Frameworks/libxcb-shape.0.dylib
       0x10b710000 -        0x10b713fff +libxcb-xfixes.0.dylib (0) <59C69D13-8E56-345B-9845-97C5C2039021> /Applications/IINA.app/Contents/Frameworks/libxcb-xfixes.0.dylib
       0x10b720000 -        0x10b723fff +libXau.6.dylib (0) <339FB33D-FFD4-3F24-86EE-4E2EC5717F99> /Applications/IINA.app/Contents/Frameworks/libXau.6.dylib
       0x10b730000 -        0x10b733ff3 +libXdmcp.6.dylib (0) <4273AC59-3C77-36C7-8FFB-B4CD4FD62056> /Applications/IINA.app/Contents/Frameworks/libXdmcp.6.dylib
       0x10b740000 -        0x10b747ff7 +libswiftos.dylib (5.0.1 - 1001.0.82.4) <A5A1203C-B3D4-350C-BC88-9CECDF62637E> /Applications/IINA.app/Contents/Frameworks/libswiftos.dylib
       0x11080e000 -        0x110c95fff +com.nvidia.web.GeForceMTLDriverWeb (10.33.0 - 10.3.3) <402C7EA4-0797-33AB-9590-CA3F90B710A6> /Library/GPUBundles/GeForceMTLDriverWeb.bundle/Contents/MacOS/GeForceMTLDriverWeb
       0x11a151000 -        0x11ab76fff +com.nvidia.web.GeForceGLDriverWeb (10.33.0 - 10.3.3) <023B1667-244E-36DB-BA98-B022943153BD> /Library/GPUBundles/GeForceGLDriverWeb.bundle/Contents/MacOS/GeForceGLDriverWeb
       0x11fc36000 -        0x11fc36fff +com.nvidia.web.GeForceVADriverWeb (10.33.0 - 10.3.3) <0AEFDE3B-6E8E-3AA9-95D6-9DE92D8096F3> /Library/GPUBundles/GeForceVADriverWeb.bundle/Contents/MacOS/GeForceVADriverWeb
       0x1211f1000 -        0x121399ff7  com.apple.audio.units.Components (1.14 - 1.14) <85C6E01E-2BF4-331D-ACA5-F528B6258318> /System/Library/Components/CoreAudio.component/Contents/MacOS/CoreAudio
    0x7fff4b548000 -     0x7fff4b896ff7  com.apple.RawCamera.bundle (8.05.0 - 1017.4.1) <937CBA8D-60C8-3739-8DFF-1C737D6339EF> /System/Library/CoreServices/RawCamera.bundle/Contents/MacOS/RawCamera
    0x7fff4f16a000 -     0x7fff4f34aff3  com.apple.avfoundation (2.0 - 1536.36) <AB868021-AB26-3198-897D-087EF3B360EA>

```

**Steps to reproduce:**

Open up any video and command+q as quick as possible, possibly within the first 5 seconds.

**How often does this happen?**

Always.

**Comment by low-batt** [comment]
This part of the crash report posted above:
```text
0   libmpv.1.dylib                	0x000000010753ec1e mpv_get_property + 14
1   com.colliderli.iina           	0x0000000106e48418 $s4IINA10PlayerCoreC6syncUIyyAC12SyncUIOptionOF + 584
2   com.colliderli.iina           	0x0000000106e480c3 $s4IINA10PlayerCoreC10syncUITimeyyF + 51
3   com.colliderli.iina           	0x0000000106d41f51 $s4IINA20MainWindowControllerC6showUI33_656F66F44299633CE9F07D3F2FD985C2LLyyF + 385
4   com.colliderli.iina           	0x0000000106d3d3b3 $s4IINA20MainWindowControllerC23windowDidExitFullScreenyy10Foundation12NotificationVF + 595
5   com.colliderli.iina           	0x0000000106d417cd $s4IINA20MainWindowControllerC15windowWillCloseyy10Foundation12NotificationVFToTm + 109
```

Identifies this as a duplicate of issue #4020. A fix in PR #4068 is under review. This is considered a high priority problem that must be fixed in the next release of IINA.

**Comment by 2V3EvG4LMJFdRe** [comment]
Closed for duplicate.
